app/src/main/java/com/willowtree/vocable/presets/PresetsViewModel.kt

```diff
@@ -0,0 +1,242 @@
+package com.willowtree.vocable.presets
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class PresetsViewModel : ViewModel() {
+
+    companion object {
+        private const val CATEGORY_GENERAL = "General"
+        private const val CATEGORY_BASIC_NEEDS = "Basic Needs"
+        private const val CATEGORY_PERSONAL_CARE = "Personal Care"
+        private const val CATEGORY_CONVERSATION = "Conversation"
+        private const val CATEGORY_ENVIRONMENT = "Environment"
+        private const val CATEGORY_FEELINGS = "Feelings"
+        private const val CATEGORY_QUESTIONS = "Questions"
+        private const val CATEGORY_TIME = "Time"
+        private const val CATEGORY_PEOPLE = "People"
+        private const val CATEGORY_NUMBERS = "Numbers"
+    }
+
+    private val categories = listOf(
+        CATEGORY_GENERAL,
+        CATEGORY_BASIC_NEEDS,
+        CATEGORY_PERSONAL_CARE,
+        CATEGORY_CONVERSATION,
+        CATEGORY_ENVIRONMENT,
+        CATEGORY_FEELINGS,
+        CATEGORY_QUESTIONS,
+        CATEGORY_TIME,
+        CATEGORY_PEOPLE,
+        CATEGORY_NUMBERS
+    )
+
+    private val generalPhrases = listOf(
```

Comment:
In the future we could put our categories and phrase lists into JSON and parse those maybe.

Reply:
Yeah I think that will definitely clean up this view model 👍